libpod/oci_conmon_linux.go

```diff
@@ -1318,6 +1318,10 @@ func (r *ConmonOCIRuntime) configureConmonEnv(ctr *Container, runtimeDir string)
 			env = append(env, e)
 		}
 	}
+	conf, ok := os.LookupEnv("CONTAINERS_CONF")
+	if ok {
```

Comment:
I haven't checked (it's almost bedtime here), but what if `CONTAINERS_CONF=` (i.e., is defined but empty)?

Reply:
I changed this from `GetEnv` to `LookupEnv` at @Luap99's suggestion to handle exactly that case.